### PR TITLE
Hide debug-kit noise routes.

### PR DESCRIPTION
**Pull request by dereuromark**
Resolves https://github.com/cakephp/debug_kit/issues/551

### CODE REVIEW

File: src/Template/Element/routes_panel.ctp [hunk 1]
```diff
@@ -15,6 +15,9 @@ $routes = Cake\Routing\Router::routes();
     </thead>
     <tbody>
     <?php foreach ($routes as $route): ?>
+        <?php if (strpos($route->getName(), 'debugkit.') === 0) {
+            continue;
```
Comment (markstory) [inline src/Template/Element/routes_panel.ctp]: Perhaps we should output them in a muted color instead of hiding entirely?

Comment (lorenzo) [reply src/Template/Element/routes_panel.ctp]: That sounds better

Comment (dereuromark) [reply src/Template/Element/routes_panel.ctp]: then u need to scroll one page. why would anyone be interested in the help tools routes? maybe have a toggle  button to include them and by default exlude? or move to the end.

Comment (ravage84) [reply src/Template/Element/routes_panel.ctp]: How about a configuration setting that is enabled or disabled by default which hides them as @dereuromark proposes?

Comment (markstory) [reply src/Template/Element/routes_panel.ctp]: Scrolling is dependent on the number of routes in the app. I don't think we can make decisions based on whether or not people will need to scroll.

Moving debugkit routes to the end is an option as well.

Comment (dereuromark) [reply src/Template/Element/routes_panel.ctp]: So are we OK with at least graying them out? #999 or sth?

Comment (markstory) [reply src/Template/Element/routes_panel.ctp]: Displaying in grey sounds good to me.

**Comment by ADmad** [comment]
@dereuromark Ping

**Comment by dereuromark** [comment]
I am fine if someone wants to finish it for us.

**Comment by saeideng** [comment]
an alternate PR merged by mars story

**Pull request closed without merging**